test/release/index.js

```diff
@@ -0,0 +1,136 @@
+/* eslint-env browser */
+/* eslint-disable prefer-arrow-callback,prefer-template */
+const mapboxgl = {};
+
+document.addEventListener('DOMContentLoaded', function() {
+    const jsProdMin = document.createElement("a");
+    jsProdMin.href = "/dist/mapbox-gl.js";
+    const css = document.createElement("a");
+    css.href = "/dist/mapbox-gl.css";
+
+    const titleElement = document.querySelector('#title');
+    const container = document.querySelector('#container');
+    const prevButton = document.querySelector('#prev');
+    const nextButton = document.querySelector('#next');
+
+    const pages = [
```

Comment:
combining all of the test json metadata into one file, with key's being these names file would make it DRY'er .